packages/basic/lib/base.js

```diff
@@ -11,6 +11,7 @@ import pino from 'pino'
 import { NonZeroExitCode } from './errors.js'
 import { cleanBasePath } from './utils.js'
 import { ChildManager } from './worker/child-manager.js'
+import { getTsconfig, ignoreDirs } from '../../node/lib/utils.js'
```

Comment:
You can't do this. You are loading a file from another package.

Reply:
What is I move them to `packages/utils`?